pallets/phala/src/mock.rs

```diff
@@ -123,6 +127,67 @@ impl stakepool::Config for Test {
 	type InsurancePeriod = MiningInsurancePeriod;
 }
 
+pub struct MockValidator;
+impl AttestationValidator for MockValidator {
+	fn validate(
+		_attestation: &Attestation,
+		_user_data_hash: &[u8; 32],
+		_now: u64,
+	) -> Result<IasFields, AttestationError> {
+		Ok(IasFields {
+			mr_enclave: [0u8; 32],
+			mr_signer: [0u8; 32],
+			isv_prod_id: [0u8; 2],
+			isv_svn: [0u8; 2],
+			report_data: [0u8; 64],
+			confidence_level: 128u8,
+		})
+	}
+}
+
+mod msg_routing {
+	use crate::mq;
```

Comment:
Ah, we may make the MessageRouteConfig generic a gain or construct the whole file with a macro.